Contents/Code/__init__.py

```diff
@@ -100,18 +112,35 @@ def Update(self, metadata, media, lang, force, movie):
         flags = flags | Prefs['hideUsefulMiscTags'] << 3 #0b01000 : Hide Useful Miscellaneous Tags
         flags = flags | Prefs['hideSpoilerTags']    << 4 #0b10000 : Hide Plot Spoiler Tags
 
+        if movie:
+            series = HttpReq("api/serie/fromep?id=%s&level=3&allpics=1&tagfilter=%d" % (aid, flags))
+            movie_episode_data = HttpReq("api/ep?id=%s" % (aid))
+
+            aid = series['id']
+
+            if movie_episode_data['name'] == 'Complete Movie':
+                movie_name = series['name']
+                movie_sort_name = series['name']
+            else:
+                movie_name = series['name'] + ' - ' + movie_episode_data['name']
+                movie_sort_name = series['name'] + ' - ' + str(movie_episode_data['epnumber']).zfill(3)
+
+            metadata.summary = summary_sanitizer(try_get(series, 'summary'))
+            metadata.title = movie_name
+            metadata.title_sort = movie_sort_name
+            metadata.rating = float(movie_episode_data['rating'])
+            year = try_get(movie_episode_data, "year", try_get(series, "year", None))
+
+            if year:
+                metadata.year = int(year)
 
-        series = HttpReq("api/serie?id=%s&level=3&allpics=1&tagfilter=%d" % (aid, flags))
+        else:
+            series = HttpReq("api/serie?id=%s&level=3&allpics=1&tagfilter=%d" % (aid, flags))
 
-        # build metadata on the TV show.
-        #metadata.summary = re.sub(LINK_REGEX, r'\1', try_get(series, 'summary'))
-        metadata.summary = summary_sanitizer(try_get(series, 'summary'))
-        metadata.title = series['name']
-        metadata.rating = float(series['rating'])
-        year = try_get(series, "year", None)
+            metadata.summary = summary_sanitizer(try_get(series, 'summary'))
+            metadata.title = series['name']
+            metadata.rating = float(series['rating'])
 
-        #if year:
```

Comment:
Any reason this was removed? even through it's commented out, it could be something we want to add back in.

Reply:
I removed it and put it inside the `if` statement of movie. The TV show model doesn't have a member called year anyway.